src/tools/rest-frontend/public/assets/translations/en.json

```diff
@@ -0,0 +1,370 @@
+    "APP.ENTRIES.SEARCH.ADVANCED.FILTERBY.TITLE": "Title",
+    "APP.ENTRIES.SEARCH.ADVANCED.FILTERBY.DESCRIPTION": "Description",
+    "APP.ENTRIES.SEARCH.ADVANCED.FILTERBY.AUTHOR": "Author",
+    "APP.ENTRIES.SEARCH.ADVANCED.FILTERBY.TAGS": "Tags",
+    "APP.ENTRIES.SEARCH.ADVANCED.SORT.ASC": "Ascending",
+    "APP.ENTRIES.SEARCH.ADVANCED.SORT.DESC": "Descending",
+    "APP.ENTRIES.SEARCH.ADVANCED.SORTBY.KEY": "Key",
+    "APP.ENTRIES.SEARCH.ADVANCED.SORTBY.TITLE": "Title",
+    "APP.ENTRIES.SEARCH.ADVANCED.SORTBY.CREATED_AT": "Creation Date",
+    "APP.ENTRIES.SEARCH.ADVANCED.SORTBY.AUTHOR": "Author",
+    "APP.ENTRIES.SEARCH.ADVANCED.SORTBY.ORGANIZATION": "Organization",
+    "APP.ENTRIES.SEARCH.ADVANCED.SORTBY.APPLICATION": "Application",
+    "APP.ENTRIES.SEARCH.ADVANCED.SORTBY.SCOPE": "Scope",
+    "APP.ENTRIES.SEARCH.ADVANCED.SORTBY.SLUG": "Slug",
+    "APP.ENTRIES.SEARCH.SEARCHBOX.BUTTON.ADVANCEDOPTIONS": "Advanced options",
+    "APP.ENTRIES.SEARCH.SEARCHBOX.BUTTON.SUBMIT": "Go!",
+    "APP.ENTRIES.SEARCH.SEARCHBOX.PLACEHOLDER": "Search for configuration snippet...",
+    "APP.ENTRIES.SEARCH.TABLE.ACTION.BUTTON.DETAILS": "Details",
+    "APP.ENTRIES.SEARCH.TABLE.HEADER.ACTIONS": "Actions",
+    "APP.ENTRIES.SEARCH.TABLE.HEADER.APPLICATION": "Application",
+    "APP.ENTRIES.SEARCH.TABLE.HEADER.AUTHOR": "Author",
+    "APP.ENTRIES.SEARCH.TABLE.HEADER.CREATEDAT": "Created at",
+    "APP.ENTRIES.SEARCH.TABLE.HEADER.KEY": "Key",
+    "APP.ENTRIES.SEARCH.TABLE.HEADER.ORGANIZATION": "Organization",
+    "APP.ENTRIES.SEARCH.TABLE.HEADER.SCOPE": "Scope",
+    "APP.ENTRIES.SEARCH.TABLE.HEADER.SLUG": "Slug",
+    "APP.ENTRIES.SEARCH.TABLE.HEADER.TITLE": "Title",
+    
+    "APP.GLOBAL.FOOTER.LINK.TITLE.SCROLLTOTOP": "Scroll to Top",
+    "APP.GLOBAL.NOTIFICATION.HEADER.CONNECTION_ISSUE": "Connection issues",
+    "APP.GLOBAL.NOTIFICATION.MESSAGE.SERVER_NOT_REACHABLE": "It seems as if the service is not reachable. This may be either due to a downtime of the service or because of your internet connection.",
+    "APP.GLOBAL.TITLE": "ElektraInitiative",
+    
+    "APP.HOME.BUTTON.EXAMPLES": "Examples",
+    "APP.HOME.BUTTON.GETSTARTED": "Get started",
+    "APP.HOME.HEADLINE.HOWITWORKS": "Want to see how it works?",
+    "APP.HOME.HEADLINE.SLOGAN.FIRST": "Configure everything, {{ adjectives[index] }}.",
+    "APP.HOME.HEADLINE.SLOGAN.SECOND": "Elektra - the configuration framework for everyone",
+    "APP.HOME.HEADLINE.WHATSNEW": "What's new?",
+    "APP.HOME.LABEL.KEYFACT.1.DETAILS": "Elektra stores configuration in a global, hierarchical key database. Mount configuration files into the hierarchy, override single configuration parameters or use profiles.",
+    "APP.HOME.LABEL.KEYFACT.1.HEADLINE": "Global configuration database",
+    "APP.HOME.LABEL.KEYFACT.2.DETAILS": "All common and important configuration formats are supported through various storage plugins - even special configurations like hosts files.",
```

Comment:
configurations  -> configuration files

Otherwise I think its easiest if I edit it after merge.